sinon/test/TestSinonSpy.py

```diff
@@ -852,3 +856,18 @@ def test240_neverCalledWithMatch_args(self):
         self.assertTrue(spy.neverCalledWithMatch("a", "e"))
         self.assertTrue(spy.neverCalledWithMatch("d", "e", "c")) #it's a combination
 
+    @sinontest
+    def test270_args(self):
```

Comment:
This additional test looks very nice, the only suggestion I have is about naming. I believe `test270_args_and_kwargs` will be more precise in this case.

Reply:
Done